configs/linode.json

```diff
@@ -0,0 +1,18 @@
+{
+    "provider": "linode",
+    "description": "Linode estimates based on July 2024 advertised prices",
+    "linodeTokenSecret": "kube-system/linode",
+    "CPU": "0.0",
```

Comment:
@Sean-Holcomb, I don't believe Linode can provide the specific pricing numbers we're looking for. I also find it challenging to see how other providers could do this because it's nearly impossible to represent CPU prices accurately in a single, static file.

In this case, the values are set to 0.0 in the config, but the provider's implementation fetches the actual pricing at runtime.

However, the Linode provider implementation currently has some limitations:
- It requires monitoring.coreos.com/v1.ServiceMonitor to function properly (see a related suggestion later).
- It does not support custom pricing. We're facing challenges with the mandatory RAM prices in OpenCost as a result.
- Cloud cost support is still under development.

An alternative solution could be adding a new function call here: https://github.com/opencost/opencost/blob/cc5773c1bc4af0838f0bc6f4a6177fe6505e12f6/pkg/costmodel/allocation_helpers.go#L1724. At this point, the node name is already known, so introducing a new function, such as NodePriceByName within the provider, should return the correct prices instead of relying on hard-coded dummy data.

What do you think of this approach?